src/contracts/markets/Abstract4626MarketWrapper.sol

```diff
@@ -190,6 +208,17 @@ contract Abstract4626MarketWrapper is Initializable, Ownable {
         emit HarvesterUpdated(_harvester);
     }
 
+    /// @notice The contract owner sets the address of the Merkle Distributor contract.
+    /// @param _merkleDistributor The address of the Merkle Distributor contract.
+    function setMerkleDistributor(address _merkleDistributor) external onlyOwner {
+        _setMerkleDistributor(_merkleDistributor);
+    }
+
+    function _setMerkleDistributor(address _merkleDistributor) internal {
+        require(_merkleDistributor != address(0), "MerkleDistributor cannot be zero address");
```

Comment:
I guess it doesn't matter setting the merkleDistributor for a lending market that doesn't use Merkle.
I do like the error messages being 31 characters or less, though. That way they fit in one EVM word (32 bytes)

Reply:
I removed the check i.e. it is possible to set merkleDistributor to address(0). 
Changed in this commit: [6cc493c](https://github.com/OriginProtocol/arm-oeth/pull/137/commits/6cc493cf65f5d225818217bb8c5770345f766053)